scripts/deployV1.1.0.ts

```diff
@@ -0,0 +1,115 @@
+import { GSN_MUMBAI_FORWARDER_CONTRACT_ADDRESS } from '@big-whale-labs/constants'
+import { OBSSStorage } from 'typechain'
+import { chains, ethereumRegex } from './helpers/data'
+import { ethers, upgrades } from 'hardhat'
+import { utils } from 'ethers'
+import { version } from '../package.json'
+import deployContact from './helpers/deployContract'
+import prompt from 'prompt'
+
+async function main() {
+  const [deployer] = await ethers.getSigners()
+
+  const provider = ethers.provider
+
+  const { chainId } = await provider.getNetwork()
+  const chainName = chains[chainId]
+
+  // Deploy the contract
+  console.log('Deploying contracts with the account:', deployer.address)
+  console.log('Using network:', chainName)
+  console.log(
+    'Account balance:',
+    utils.formatEther(await deployer.getBalance())
+  )
+
```

Comment:
Let's make 2 prompts here:
```TS
  const { isProduction } = await prompt.get({ properties: { isProduction: required: true, type: 'boolean', default: false}})
     
     const {defaultObssProxy, ...} = isProducton ? { ... } : { ... }
      
const {
    forwarder,
    ketlTeamTokenId,
    obssProxyAddress,
    profilesContractAddress,
    feedsContractAddress,
  } = await prompt.get({
    properties: { ..., 
          obssProxyAddress: {
        required: true,
        message: 'OBSS Proxy address',
        pattern: ethereumRegex,
        default: defaultObssProxy,
      },
      ...
      }
 ```

Reply:
So we don't need to copy-paste from the script

Reply:
Resolved.